src/semgrep_agent/main.py

```diff
@@ -87,13 +88,25 @@ def main(
     # Setup URL/Token
     sapp = Sapp(url=publish_url, token=publish_token, deployment_id=publish_deployment)
     maybe_print_debug_info(meta)
-    sapp.report_start(meta)
+    policy = sapp.report_start(meta)
     if sapp.is_configured:
         click.echo(
-            f"| semgrep.dev - logged in as deployment #{sapp.deployment_id}", err=True,
+            f"| {server} - logged in as deployment #{sapp.deployment_id}", err=True,
         )
+        if policy:
+            click.echo(f"| using policy {policy}")
     else:
-        click.echo("| semgrep.dev - not logged in", err=True)
+        click.echo(f"| {server} - not logged in", err=True)
+
+    for env_var in [
+        "SEMGREP_REPO_URL",
+        "SEMGREP_JOB_URL",
+        "SEMGREP_BRANCH",
+        "SEMGREP_PR_ID",
+        "SEMGREP_PR_TITLE",
+    ]:
+        if os.getenv(env_var):
+            click.echo(f"| {env_var}={os.getenv(env_var)}")
```

Comment:
```suggestion
            click.echo(f"| {env_var} - {os.getenv(env_var)}")
```

Just keeping the same format as existing lines.

Reply:
I don't feel _that_ strongly about this, but I do think it looks wrong in the case of environment variables to use a dash instead of equals. I prefer the consistency with how they are set in the terminal.

Reply:
Fair enough!